.circleci/config.yml

```diff
@@ -263,37 +264,35 @@ jobs:
             (cd examples/stm32f469_discovery/blink; lbuild -D ::generate_module_docs=True build -m "modm:**")
             (cd docs/src/reference && ln -s ../../../examples/stm32f469_discovery/blink/modm/docs/module)
             (cd docs && mkdocs build)
+      - persist_to_workspace:
+          root: docs
+          paths:
+            - modm.io
 
-  build-docs-upload:
+  upload-docs:
     docker:
       - image: modm/modm-build:latest
     steps:
       - checkout
+      - attach_workspace:
+          at: /tmp/docs
       - add_ssh_keys:
           fingerprints:
             - "8c:e7:b6:51:2a:40:66:e6:e9:50:ea:fe:38:49:1d:3f"
       - run:
+          name: Clone Old Docs
           command: |
-            (git submodule sync && git submodule update --init --jobs 8) &
-            pip3 install -U lbuild &
-            git clone git@github.com:modm-ext/modm.io.git docs/modm.io &
-            wait
+            git clone --depth 1 git@github.com:modm-ext/modm.io.git docs/modm.io
+            cp -R /tmp/docs/modm.io/ docs/
```

Comment:
Does this delete files?

Reply:
You are right. We should delete the directory (except the .git subdir) before, like
```
find -not -path "./.git/*" -not -name ".git" -delete
```
Or do the checkout step in the `build-docs` step.

Reply:
The easy way is the `--no-checkout` option of git ...